README.md

```diff
@@ -1 +1,64 @@
-# Book-AwesomES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books-ES6
+
+The aim of this project is to restructure the Awesome books app code by using ES6 syntax and organising the workspace using modules.
+
+The code is divided into modules and each module is saved in a separate file in the `modules` dir. and imported in the `index.js` file.
+
+## Live Demo
+
+[link to live demo]()
+
+> Awesome-Books-ES6: Refractor to use JavaScript modules and ES6 Syntax.
+
+## Built With
+
+- JavaScript
+- HTML
+- CSS
+
+## Getting Started
+
+### Prerequisites
+
+- In the repository, copy the `SSH` code snippet for Awesome-book-2
+- clone into the repository by using `git clone` in the git bash terminal
+- cd into the repository `cd Awesome-Books-ES6`
+- open VScode terminal `code .`
+- Go live (live server)
+
+### Setup
+
+- Linters
+- Git bash (optional: vscode gitbash terminal)
+
+### Install
+
+- Live server on VScode
+
+## Author
+
+👤 **Rejoice Evans**
+
+- GitHub: [@githubhandle]( https://github.com/rmjspecial)
+- twitter:  [@twitterhandle](https://twitter.com/rmjspecial2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rejoice-evans-74882122a/)
+
+## 🤝 Contributing
+
```

Comment:
- [ ] [optional] It would be nice to add a link to your issue page, where contributions could be made.